onnxruntime/core/mlas/lib/convsym.cpp

```diff
@@ -191,8 +191,8 @@ MlasConvSymSetOutputZeroPoint(
     bool InputIsSigned
     )
 {
-    constexpr int32_t minimum = InputIsSigned ? std::numeric_limits<int8_t>::min() : std::numeric_limits<uint8_t>::min();
-    constexpr int32_t maximum = InputIsSigned ? std::numeric_limits<int8_t>::max() : std::numeric_limits<uint8_t>::max();
+    int32_t minimum = InputIsSigned ? std::numeric_limits<int8_t>::min() : std::numeric_limits<uint8_t>::min();
```

Comment:
different type ? compiler are all ok with such ?

Reply:
yes, it is valid. Here is a list of valid cases: https://en.cppreference.com/w/c/language/operator_other#Conditional_operator